src/java/main/org/apache/zookeeper/server/NettyServerCnxn.java

```diff
@@ -71,7 +71,7 @@
     NettyServerCnxnFactory factory;
     boolean initialized;
     
-    NettyServerCnxn(Channel channel, ZooKeeperServer zks, NettyServerCnxnFactory factory) {
+    public NettyServerCnxn(Channel channel, ZooKeeperServer zks, NettyServerCnxnFactory factory) {
```

Comment:
@hanm - >>> so all tests fall back to create NIOServerCnxnx instead
It is not the case, they don't fall back to NIO - they fail.
Here is printout form running tests with both Netty and MockNettyServerCnxn not having public ctor:
   [junit] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.159 sec
  **  [junit] Test org.apache.zookeeper.server.NettyServerCnxnTest FAILED **
    [junit] Running org.apache.zookeeper.server.PrepRequestProcessorTest
    [junit] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.279 sec
    [junit] Running org.apache.zookeeper.server.PurgeTxnTest
    [junit] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.431 sec
    [junit] Running org.apache.zookeeper.server.ReferenceCountedACLCacheTest
    [junit] Tests run: 10, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.089 sec
    [junit] Running org.apache.zookeeper.server.SerializationPerfTest
    [junit] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.114 sec
    [junit] Running org.apache.zookeeper.server.ServerCxnExceptionsTest
    [junit] Tests run: 6, Failures: 0, Errors: 6, Skipped: 0, Time elapsed: 0.284 sec
**    [junit] Test org.apache.zookeeper.server.ServerCxnExceptionsTest FAILED **